src/main/java/com/bandwidth/sqs/queue/RedrivePolicy.java

```diff
@@ -0,0 +1,23 @@
+package com.bandwidth.sqs.queue;
+
+
+import org.immutables.value.Value.Immutable;
+
+import java.text.MessageFormat;
+
+@Immutable
+public interface RedrivePolicy {
```

Comment:
This is what AWS SQS uses to refer to dead-letter queues. Once a message has been 'received' the maximum number of times from the same queue, if it is then requeued, it will go to the dead-letter queue instead of that queue